net-im/toxic/toxic-0.8.2.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1 xdg
+
+DESCRIPTION="A curses-based client for Tox"
+HOMEPAGE="https://github.com/JFreegman/toxic"
+PATCHES_GIST="35e877418de56afd7305dd55c89b536e"
+PATCHES_REV="fe985af45f1afe73f1e79bc1ffb03c6ac2d25c88"
+SRC_URI="https://github.com/JFreegman/toxic/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		 https://gist.github.com/fx-carton/${PATCHES_GIST}/archive/${PATCHES_REV}.zip -> ${P}_patches.zip"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+X +audio notifications +python +qrcode +video"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+# Not a typo; net-libs/tox only has a 'both or neither' option
+RDEPEND="
+	audio? ( net-libs/tox:0/0.2[av] )
+	video? ( net-libs/tox:0/0.2[av] )
+	net-libs/tox:0/0.2
+	dev-libs/libconfig
+	net-misc/curl:0=
+	sys-libs/ncurses:0=
+	audio? (
+		media-libs/openal
+		media-libs/freealut
+	)
+	video? (
+		media-libs/libvpx:=
+		x11-libs/libX11
+	)
+	notifications? ( x11-libs/libnotify )
+	python? ( ${PYTHON_DEPS} )
+	qrcode? ( media-gfx/qrencode )
+"
+DEPEND="
+	virtual/pkgconfig
+	${RDEPEND}
+"
+
+PATCHES=(
+	"${WORKDIR}/${PATCHES_GIST}-${PATCHES_REV}/${P}-make-qrencode-optional.patch"
+	"${FILESDIR}/${P}-verbose-build-log.patch"
+	)
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+	[ "${PV}" = "0.8.2" ] && sed -i \
```

Comment:
Always `[[ ]]`, and then you don't need quoting. Also, I told you should **die** when PV changes, not silently ignore the sed.

Reply:
Why a regex comparison? A string comparison is what we want here...

Reply:
It's not regex comparison. It's bash testing thingie that doesn't suffer from all the inconveniences of external command.

Reply:
I meant pattern matching, my bad.
But I see the point about avoiding an external command.